src/jit/compiler.cpp

```diff
@@ -2716,8 +2713,11 @@ void Compiler::compSetProcessor()
             codeGen->getEmitter()->SetContainsAVX(false);
             codeGen->getEmitter()->SetContains256bitAVX(false);
         }
-        else if (CanUseSSE4())
+        else if (compSupports(InstructionSet_SSE41) || compSupports(InstructionSet_SSE42))
```

Comment:
We could create a `compSupportsAny(InstructionSet_SSE41 | InstructionSet_SSE42)` function to simplify this, if we think it is worthwhile.